src/nvim/sign_defs.h

```diff
@@ -9,17 +9,18 @@ typedef struct signlist signlist_T;
 
 struct signlist
 {
-    int id;             /* unique identifier for each placed sign */
-    linenr_T lnum;      /* line number which has this sign */
-    int typenr;         /* typenr of sign */
-    signlist_T *next;   /* next signlist entry */
+    int id;             // unique identifier for each placed sign
+    linenr_T lnum;      // line number which has this sign
+    int typenr;         // typenr of sign
+    signlist_T *next;   // next signlist entry
 };
 
-/* type argument for buf_getsigntype() */
-#define SIGN_ANY	0
-#define SIGN_LINEHL	1
-#define SIGN_ICON	2
-#define SIGN_TEXT	3
+// type argument for buf_getsigntype()
+#define SIGN_ANY     0
+#define SIGN_LINEHL  1
+#define SIGN_ICON    2
+#define SIGN_TEXT    3
+#define SIGN_NUMHL   4
```

Comment:
might be a good time to make this an enum.

Reply:
Done, although I'm not sure what the naming convention for enums is supposed to be. It's not so consistent.